### PR TITLE
[docker] clean docker images for centos and ubuntu

### PR DESCRIPTION
**Pull request by fabiencastan**
Same PR than https://github.com/alicevision/AliceVision/pull/853 with a fix to be able to merge.

### CODE REVIEW

**Review by servantf — APPROVED** [review]

File: Dockerfile_deps [hunk 1]
```diff
@@ -1,110 +0,0 @@
...
-# docker run -it --runtime=nvidia alicevision
-
-
-# OS/Version (FILE): cat /etc/issue.net
-# Cuda version (ENV): $CUDA_VERSION
-
-ENV AV_DEPS_DEV=/opt/AliceVisionDeps_git \
-    AV_DEPS_BUILD=/tmp/AliceVisionDeps_build \
-    AV_INSTALL=/opt/AliceVision_install \
-    AV_BUNDLE=/opt/AliceVision_bundle \
```
Comment (fabiencastan) [inline Dockerfile_deps:23]: @deece Why did you remove AV_BUNDLE?

File: docker/Dockerfile_centos [hunk 2]
```diff
@@ -17,28 +17,23 @@ LABEL maintainer="AliceVision Team alicevision-team@googlegroups.com"
...
-\
-make bundle && \
-\
-cd /opt && \
-\
-rm -rf "${AV_BUILD}"
+    make -j"$(nproc)" && \
+    make install && \
+    rm -rf "${AV_DEV}" "${AV_BUILD}" && \
+    echo "export ALICEVISION_SENSOR_DB=${AV_BUNDLE}/share/aliceVision/cameraSensors.db" >> /etc/profile.d/alicevision.sh
```
Comment (fabiencastan) [inline docker/Dockerfile_centos:37]: and AV_BUNDLE is still used here

File: Dockerfile_deps (hunk 1 above)
Comment (fabiencastan) [reply Dockerfile_deps:23]: We need it on the Meshroom side.

File: CMakeLists.txt [hunk 3]
```diff
@@ -236,10 +251,12 @@ endif()
 
 if(AV_BUILD_SUITESPARSE)
 set(SUITESPARSE_TARGET suitesparse)
-set(SUITESPARSE_INTERNAL_MAKE_CMD LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR} make BLAS="${BLAS_LIBRARIES}" LAPACK="${LAPACK_LIBRARIES}")
+set(SUITESPARSE_INTERNAL_MAKE_CMD LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR} $(MAKE) BLAS="${BLAS_LIBRARIES}" LAPACK="${LAPACK_LIBRARIES}")
 ExternalProject_Add(${SUITESPARSE_TARGET}
-       # URL https://github.com/DrTimothyAldenDavis/SuiteSparse/archive/v4.5.6.tar.gz
-       URL https://github.com/DrTimothyAldenDavis/SuiteSparse/archive/v5.7.2.tar.gz  # requires gxx >= 4.9, centos 7 use gxx-4.8.5 by default
+       # URL https://github.com/DrToimothyAldenDavis/SuiteSparse/archive/v4.5.6.tar.gz
```
Comment (fabiencastan) [inline CMakeLists.txt:256]:
```suggestion
       # URL https://github.com/DrTimothyAldenDavis/SuiteSparse/archive/v4.5.6.tar.gz
```

File: Dockerfile_deps (hunk 1 above)
Comment (deece) [reply Dockerfile_deps:23]: It didn't seem to be needed - Meshroom was happy to run from the install, and we need the install to build Meshroom (the bundle is missing some components needed for compilation)

File: docker/Dockerfile_centos (hunk 2 above)
Comment (deece) [reply docker/Dockerfile_centos:42]: That's a mistake, it should point to AV_INSTALL

**Comment by deece** [comment]
@fabiencastan I think that the images built from here should be pushed to Dockerhub, probably for every release, but until the next release, an interim snapshot of develop would be nice.

**Comment by fabiencastan** [comment]
Hi @deece,
Yes, it's done for each release. If there is no new issue discovered, the next release should probably be next week.
